dotnet/src/Functions/Functions.OpenApi/Model/RestApiInfo.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Plugins.OpenApi;
+
+/// <summary>
+/// The REST API information.
+/// </summary>
+public sealed class RestApiInfo
+{
+    /// <summary>
+    /// The title of the application.
+    /// </summary>
+    public string? Title { get; init; }
+
+    /// <summary>
+    /// A short description of the application.
+    /// </summary>
+    public string? Description { get; init; }
+
+    /// <summary>
+    /// REQUIRED. The version of the OpenAPI document.
```

Comment:
If anything here is required, should we add a constructor that requires these to be provided?

Reply:
Updated the description, it's not required.